qa/rpc-tests/test_framework/util.py

```diff
@@ -267,7 +270,7 @@ def rebuild_cache():
                 args.append("-connect=127.0.0.1:"+str(p2p_port(0)))
             bitcoind_processes[i] = subprocess.Popen(args)
             if os.getenv("PYTHON_DEBUG", ""):
-                print("initialize_chain: bitcoind started, waiting for RPC to come up")
+                print("initialize_chain: %s started, waiting for RPC to come up" % zcashd_binary())
```

Comment:
```suggestion
                print("initialize_chain: %s started, waiting for RPC to come up" % (zcashd_binary(),))
```
Style nit: always using a tuple as the right argument of `%` leads to more regular behaviour that is less likely to hide type errors. (`%` with an untupled value can blow up at runtime if the argument turns out to be a tuple with more elements than there are placeholders in the format string.)